mobile/ios/OriginMarketplace/Info.plist

```diff
@@ -48,7 +48,7 @@
 	<key>NSLocationAlwaysUsageDescription</key>
 	<string>We will not be using the location</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string>We will not be using the location</string>
+	<string>We would like to determine listing&apos;s location. </string>
```

Comment:
Maybe a bit more verbose since Apple has been so annoying about these:

`We would like to use your location to add location data to any listings that you create`

Reply:
That sounds better thanks!